### ISSUE TITLE
Multiple targets & values

### ISSUE THREAD

**Issue by qasim** [WatchBeam/split-html-loader#1]
I love this project, especially as a lightweight alternative to clunky templating engines used just to accomplish basic splitting.

Could it accept multiple values by taking in a query object instead of individually assigning a `target` and `value`?

The current example would then be like:

```js
module.exports = {
  // ...
  module: {
    preLoaders: [{
      test: /\.html$/,
      loader: 'split-html',
      query: {
        platform: 'mobile',
        locale: 'en_ca'
      }
    }],
    loaders: [{ test: /\.html$/, loader: 'html' }]
  }
};
```

```js
fs.writeFileSync('./index.html', split.string(myHTML, {
  platform: 'mobile',
  locale: 'en_ca'
}));
```

If you think this is a good idea for the project, I'd love to give it a try. ☺️

**Comment by connor4312** [comment]
Sure, happy to accept a PR for that 👍

**Comment by connor4312** [comment]
Resolved in #2
